remoteappmanager/docker/container_manager.py

```diff
@@ -127,8 +127,67 @@ def containers_from_mapping_id(self, user_name, mapping_id):
             'label': ['{0}={1}'.format(k, v) for k, v in labels.items()]
         }
 
+        containers = yield self.containers_from_filters(filters=filters)
+
+        return containers
+
+    @gen.coroutine
+    def container_from_url_id(self, url_id):
```

Comment:
(layout): This function does not have to be 12 lines of code. I would prefer that we use empty lines inside functions only if necessary. 

For example the following:

```
labels = {SIMPHONY_NS+"url_id": url_id,}
filters = {
       'label': ['{0}={1}'.format(k, v) for k, v in labels.items()]
}
containers = yield self.containers_from_filters(filters=filters)
return containers[0] if len(containers) else None
```

is equally understandable.